tests/e2e_args.py

```diff
@@ -114,6 +114,12 @@ def cli_args(add=lambda x: None, parser=None, accept_unknown=False):
         help="Domain name used for node certificate verification",
         default="ccf.io",
     )
+    parser.add_argument(
+        "--participants-curve",
+        help="Curve to use for member and user identities (e.g. secp384r1)",
```

Comment:
Minor: Can we call this "_Default_ curve", and add a test (presumably in `memberclient.py`) of deliberately using a mix of curves for different members, to ensure that works correctly?

Reply:
The new member that is added in the `memberclient` test is now on a different curve, so that we test this on every CI run.